src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        Random random = new Random();
+        Ball bullet = new Ball();
+        bullet.setNumber(random.nextInt(100));
+        bullet.setColor(color.getRandomColor());
```

Comment:
Create a constructor in the Ball class that can accept these two arguments.
Make 101 a class constant, use CONSTANT_CASE (101 if we want to follow the requirements precisely, now the maximum possible number will be 99, look up how nextInt works - passed value is excluded)